snowexsql/tables/doi.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Column, String, Integer, ForeignKey, Date
+from sqlalchemy.orm import relationship, declared_attr
+
+from .base import Base
+
+
+class DOI(Base):
+    """
+    Table stores DOI values for each unique publication
+    """
+    __tablename__ = 'dois'
+
+    id = Column(Integer, primary_key=True)
+    doi = Column(String())
+
+
+class DOIBase:
```

Comment:
This is getting into the weeds of choosing name.
My suggestion would be to call this `HasOne_DOI` and then follow this a convention for all other classes like this.

Would read nicely for instance when you use that in the later example:
```python
class ImageData(Base, Measurement, HasOne_DOI):
```

It is a bit verbose, but I prefer that over ambiguity.
Thoughts?